content/projects/gsoc/proposing-project-ideas.adoc

```diff
@@ -128,18 +140,110 @@ link:https://plugins.jenkins.io/role-strategy[Role Strategy Plugin]
 link:../../students[Information page for students]
 
 
-==== Newbie-friendly issues
+== Newbie-friendly issues
 
 If you are a potential mentor, propose examples of tickets the applicants could
 study while preparing their project proposals.
 We do NOT require students to make contributions before applying,
 but such tasks may help to select students who are interested to work on the project.
+```
+
+=== Using a Google Doc for capturing the details of the idea
```

Comment:
@oleg-nenashev The sub-level for this section goes with line 93.
```
Using `.adoc` for capturing the details of the idea`
```
It is not obvious but line 149 closes the triple-single-quote that started on line 98 delimiting the template example. The `Newbie-friendly issues` is actually inside that template.

Reply:
Ah right, sorry